tests/queries/test_bulk_update.py

```diff
@@ -125,6 +125,7 @@ def test_invalid_batch_size(self):
         msg = "Batch size must be a positive integer."
         with self.assertRaisesMessage(ValueError, msg):
             Note.objects.bulk_update([], fields=["note"], batch_size=-1)
+            Note.objects.bulk_update([], fields=["note"], batch_size=0)
```

Comment:
This line is never executed.
Instead:
```
        with self.assertRaisesMessage(ValueError, msg):
            Note.objects.bulk_update([], fields=["note"], batch_size=0)
```

Reply:
> But I'd be in favour of raising an error in this case +1

Yep, I prefer an error too. Note: The same file has `bulk_create()` and it raises an error if `batch_size=0`.
Also, the error message won't be consistent with the behavior when a user explicitly passes `batch_size=0` (Intentionally, or, by mistake :D).

> This line is never executed.

Thank you, I'll update it.